uiowa/uiowa
# Facilities - pull Building Coordinators from API and display on building pages

Relates to #7200 

# How to test

- `ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local fm-buildings`
- Compare a few random buildings results to the info in the API (https://bizhub.facilities.uiowa.edu/bizhub/ext/bldgCoordinators) OR find the building on the current facilities site (https://www.facilities.uiowa.edu/buildings). Note: most buildings won't have all five possible coordinators and two possible managers
- Check a building with no coordinators or managers, like 7 Triangle Place, and make sure no data is in those fields.
- UIHC - Des Moines Administrative Office is another good one to check, should have one coordinator and no managers
- Delete the coordinator/manager data for one buildings and change another one to fake values. Run the fm-buildings command again. (I change data for the first couple buildings so I can ctrl+C the command after a minute instead of waiting twelve minutes). Check the buildings, they should be back to the correct data.

## Building Coordinators View

- On any page, place the Building Coordinators List view on the page.
- It should render the columns similar to https://www.facilities.uiowa.edu/building-coordinators
- The filter should search for each of the columns, but for the contacts column, just the names, not email/phone/etc.
